__tests__/models/form.test.ts

```diff
@@ -0,0 +1,113 @@
+import { Application, Resource, Form } from '@models';
+import { faker } from '@faker-js/faker';
+import { status } from '@const/enumTypes';
+
+/**
+ * Test Form Model.
+ */
+describe('Form models tests', () => {
+  test('test with correct data', async () => {
+    for (let i = 0; i < 1; i++) {
+      const formName = faker.word.adjective();
+      const saveResData = await new Resource({
+        name: formName,
+      }).save();
+
+      const application = await Application.findOne();
```

Comment:
@ketanreliefapps an application must exist when doing the .findOne
It can be the case sometimes, but as the tests can be triggered in any order, you will find cases where the application does not exist
and so, the CI tests can fail, as they did after last push

Reply:
@AntoineRelief , can I create new Application?

Reply:
@ketanreliefapps Yes you should create a new application so we're sure there is at least one.